homeworks/hw2_app/management/commands/creating_orders.py

```diff
@@ -0,0 +1,22 @@
+from django.core.management import BaseCommand
+from django.utils import lorem_ipsum
+from ...models import  Order,Client,Product
+
+
+class Command(BaseCommand):
+	help = "Создаем базу наших клиентов"
+
+	def handle(self, *args, **options):
+		clients = Client.objects.all()
+		products = Product.objects.all()
+		random_count_of_products = __import__("random").randrange(2,21)
+
+
+		for client in clients:
+			__import__("random").choice(products)
+			for i in range(1,random_count_of_products):
+				order = Order(
+					customer_order=client,
+					# product_order=__import__("random").choice(products),
+					)
+				# order.save()
```

Comment:
Можно сделать так
```suggestion
                #  по сути можно создать заказ только с теми полями, которые не авто и не ManyToManyField
                order = Order(customer_order=choice(clients), total_amount_of_the_order=0)
                order.save()

		for _ in range(10):
                    product = choice(products)
                    
                    #  добавляем заказы, обращаясь через поле которое имеет ManyToManyField
                    order.customer_order.add(product) # можно еще использовать set, чтобы добавлять набор заказов
                    order.total_amount_of_the_order+= product.price
```
Возможно где-то ошибся, но логика должна быть верной